<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup_for_event_Dialog</class>
 <widget class="QDialog" name="Signup_for_event_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="EventsTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>581</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="End_PB">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zakończ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Signup_PB">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zapisz się</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Search_PB">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Szukaj</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Coach_CB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Godzina od</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="Time_begin_TE">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Miejsce</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Dzień</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Place_CB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prowadzący</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Day_CB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="Time_end_TE">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Godzina do:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Znajdź pasujące zajęcia</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
